hw12/hw12_1.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+try:
+    with open('json') as file:
```

Comment:
if you expect json data in file - name it something like `phonebook.json` using json extention